frontend/src/components/NavBar/PrintImage/form.tsx

```diff
@@ -0,0 +1,650 @@
+  IconButton,
+  Menu,
+  MenuItem,
+  Switch,
+  TextField,
+  Theme,
+  Typography,
+  WithStyles,
+  createStyles,
+  makeStyles,
+  withStyles,
+} from '@material-ui/core';
+import { GetApp, Cancel, VisibilityOff } from '@material-ui/icons';
+import React, { useCallback } from 'react';
+import ToggleButtonGroup from '@material-ui/lab/ToggleButtonGroup';
+import ToggleButton from '@material-ui/lab/ToggleButton';
+import { cyanBlue } from 'muiTheme';
+import { SimpleBoundaryDropdown } from 'components/MapView/Layers/BoundaryDropdown';
+import { AdminCodeString } from 'config/types';
+import { useSafeTranslation } from '../../../i18n';
+import { MapDimensions, Toggles } from './printImage.types';
+
+interface ToggleSelectorProps {
+  title: string;
+  value: number;
+  options: {
+    value: number;
+    comp:
+      | React.JSX.Element
+      | (({ value }: { value: number }) => React.JSX.Element);
+    disabled?: boolean;
+  }[];
+  iconProp?: number;
+  align?: 'start' | 'end';
+  setValue: (v: number) => void;
+}
+
+const toggleSelectorStyles = makeStyles(() => ({
+  wrapper: {
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+    '& h4': {
+      fontSize: '14px',
+    },
+  },
+  buttonGroup: { display: 'flex' },
+  button: {
+    height: '32px',
+    width: '36px',
+    padding: '4px',
+    fontSize: '0.8rem',
+    borderLeft: '1px solid rgba(0, 0, 0, 0.12) !important',
+  },
+}));
+
+function ToggleSelector({
+  title,
+  options,
+  value,
+  iconProp,
+  align,
+  setValue,
+}: ToggleSelectorProps) {
+  const classes = toggleSelectorStyles();
+  return (
+    <div className={classes.wrapper}>
+      <Typography variant="h4" style={{ textAlign: align }}>
+        {title}
+      </Typography>
+      <ToggleButtonGroup
+        value={value}
+        exclusive
+        className={classes.buttonGroup}
+        style={{ justifyContent: align }}
+      >
+        {options.map(x => (
+          <ToggleButton
+            key={x.value}
+            className={classes.button}
+            value={x.value}
+            onClick={() => {
+              setValue(x.value);
+            }}
+            disabled={x.disabled}
+          >
+            {typeof x.comp === 'function' && typeof iconProp === 'number' ? (
+              <x.comp value={iconProp} />
+            ) : (
+              x.comp
+            )}
+          </ToggleButton>
+        ))}
+      </ToggleButtonGroup>
+    </div>
+  );
+}
+
+// The SectionToggle contain a toggle, a label, and (when opened) a container that animates open
+function SectionToggle({
+  title,
+  children,
+  expanded,
+  handleChange,
+  classes,
+}: {
+  title: string;
+  children?: React.ReactNode;
+  expanded: boolean;
+  handleChange: (
+    event: React.ChangeEvent<HTMLInputElement>,
+    checked: boolean,
+  ) => void;
+} & WithStyles<typeof styles>) {
+  return (
+    <>
+      <div className={classes.collapsibleWrapper}>
+        <Switch
```

Comment:
I would suggest using the switch we use for the layers as a shared component, and update its design to match the figma.

Reply:
<img width="277" alt="Screenshot 2024-06-12 at 12 12 27 PM" src="https://github.com/WFP-VAM/prism-app/assets/16843267/5c8bdf19-a41f-4791-8b0e-f52847514737">

Reply:
Good call. I've created a shared switch component in components/common that's now used in both places